woocommerce/woocommerce
# Analyzer CLI: Add other schemas to scan

### All Submissions:






### Changes proposed in this Pull Request:

Ananlyzer CLI tool scans the database schema for changes between branches so they may be surfaced on pull requests. This PR adds two other places where schemas are declared, `OrdersTableDataStore` and `ProductAttributesLookup` classes.

### How to test the changes in this Pull Request:

1. Introduce a schema change to any, all, or some of the following places. Commit your changes locally and be sure to have `trunk` updated to the latest.
https://github.com/woocommerce/woocommerce/blob/e4824b9660e06dbcf65de3e46231275f9b606701/plugins/woocommerce/includes/class-wc-install.php#L1014
https://github.com/woocommerce/woocommerce/blob/e4824b9660e06dbcf65de3e46231275f9b606701/plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php#L828
https://github.com/woocommerce/woocommerce/blob/e4824b9660e06dbcf65de3e46231275f9b606701/plugins/woocommerce/src/Internal/ProductAttributesLookup/DataRegenerator.php#L473
2. Run the CLI tool `./tools/code-analyzer/bin/dev analyzer add/products-orders-dbschema-scan`
3. See the resulting schema changes picked up.
```
## SCHEMA CHANGES
---------------------------------------------------
 NOTICE | Schema changes detected in WooCommerce Base Schema as of 6.6.0
---------------------------------------------------
 NOTICE | Schema changes detected in OrdersTableDataStore Schema as of 6.6.0
---------------------------------------------------
 NOTICE | Schema changes detected in ProductAttributesLookup Schema as of 6.6.0
---------------------------------------------------
```

### Other information:




### FOR PR REVIEWER ONLY:
